src/WebJobs.Script.WebHost/Management/InstanceManager.cs

```diff
@@ -31,7 +31,7 @@ public InstanceManager(ScriptSettingsManager settingsManager, WebHostSettings we
             _client = client;
         }
 
-        public bool StartAssignment(HostAssignmentContext assignmentContext)
+        public bool Assign(HostAssignmentContext context)
```

Comment:
Should this be renamed back to `StartAssignment` since the behavior was switched back to a fire and forget?

Reply:
Yes, I'll do that